wzdx/raw_to_standard/planned_events.py

```diff
@@ -64,15 +73,25 @@ def generate_standard_messages_from_string(input_file_contents):
     return standard_messages
 
 
+# TODO: Integrate Category
+def is_incident(msg):
+    id = msg.get('properties', {}).get('id', '')
+    type = msg.get('properties', {}).get('type', '')
+    # category = msg.get('properties', {}).get('Category')
+    is_incident = re.match(INCIDENT_ID_REGEX, id) != None
+    # is_wz = WORK_ZONE_INCIDENT_TYPES.get(type, {}).get(category)
+    is_wz = WORK_ZONE_INCIDENT_TYPES.get(type, {}) != {}
+    return is_incident, is_wz
+
+
 def generate_raw_messages(message_string):
     msg = json.loads(message_string)
     messages = []
 
     separated_messages = expand_event_directions(msg)
```

Comment:
I removed it because everything it handles was already handled by the schema validation (from confluent) completed in the GCP dataflow. Didn't really think about uses other than GCP, can add it back if you want
Removed loop